### PR TITLE
 fix/(docs/website) add missing labels

### PR DESCRIPTION
**Pull request by anna-lach**
#211

### CODE REVIEW

File: docs/website/src/site/_includes/search.njk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<input id="searchField" class="ds-sidebar-nav__search" type="search" placeholder="Search" />
+<input id="searchField" class="ds-sidebar-nav__search" type="text" placeholder="Search" aria-label="Search the documentation website content"/>
```
Comment (Diaan) [inline docs/website/src/site/_includes/search.njk:1]: why did you change the type from `search` to `text`?

**Review by Diaan — APPROVED** [review]

File: docs/website/src/site/_includes/search.njk (hunk 1 above)
Comment (anna-lach) [reply docs/website/src/site/_includes/search.njk:1]: due to Michael's suggestion :) https://sanoma.slack.com/archives/C03SA9HUUA3/p1674213916908739
